src/modules/user/repository/user.repository.ts

```
@@ -93,33 +101,43 @@ export class UserRepository {
 
   async updatePassword(id, password: string): Promise<UsersEntity> {
-    const user = await this.usersRepository.findOneBy({id}).catch(handleError);
+    const user = await this.usersRepository
+      .findOneBy({ id })
+      .catch(handleError);
     const data = {
       recoverPasswordToken: null,
       password,
     };
 
     delete user.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)